### PR TITLE
Ai 12 place item reserve

### PR DESCRIPTION
**Pull request by trulyVolodymyr**
added place item component with date and guest amount components

### CODE REVIEW

File: src/components/header/FiltersButton.vue [hunk 1]
```diff
@@ -13,5 +13,6 @@ const { filtersModalVisability } = storeToRefs(filterStore)
 
 function showFiltersModal () {
   filtersModalVisability.value = true
+  console.log(filtersModalVisability.value)
```
Comment (ghost) [inline src/components/header/FiltersButton.vue]: remove this console.log

File: src/views/place-item/components/PlaceItemReserve.vue [hunk 2]
```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class="ml-auto shadow-2xl p-2 h-[max-content]">
+    <p class="mb-4">
+      <span
+        class="font-bold text-lg"
+      >
+        $ {{ place.pricing.rate.amount }}
+      </span> night
+    </p>
```
Comment (ghost) [inline src/views/place-item/components/PlaceItemReserve.vue]: in one line pls

File: src/views/place-item/components/PlaceItemReserve.vue [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    >
+      <div class="flex justify-between mb-2">
+        <p class="text-sm">${{ place.pricing.rate.amount }} x {{ numberOfDays }} nigths </p>
+        <p class="text-sm">${{ totalSum }}</p>
+      </div>
+
+      <div class="flex justify-between">
+        <p class="text-sm">
+          Service fee
+        </p>
```
Comment (ghost) [inline src/views/place-item/components/PlaceItemReserve.vue]: one line

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/components/header/FiltersButton.vue (hunk 1 above)
Comment (trulyVolodymyr) [reply src/components/header/FiltersButton.vue]: done

File: src/views/place-item/components/PlaceItemReserve.vue (hunk 2 above)
Comment (trulyVolodymyr) [reply src/views/place-item/components/PlaceItemReserve.vue]: done

File: src/views/place-item/components/PlaceItemReserve.vue (hunk 3 above)
Comment (trulyVolodymyr) [reply src/views/place-item/components/PlaceItemReserve.vue]: done

**Review by ghost — APPROVED** [review]

**Pull request merged**